minecraft_servers/ikara/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+  "server_name": "ikara",
+  "nice_name": "Ikara",
+  "direct_ip": "ikara.gg",
+  "server_wildcards": [
+    "%.ikara.gg"
+  ],
+  "supported_languages": [
+     "en"
+  ],
+  "social": {
+    "web": "https://ikara.gg",
+    "web_shop": "https://store.ikara.gg",
+    "web_support": "https://ikara.gg/discord",

```

Comment:
`web_support` cannot be a discord server or forum; use a contact form, faq or similar or remove